kernel/kernel-api/src/main/java/io/delta/kernel/internal/TableImpl.java

```diff
@@ -358,4 +358,8 @@ private void recordSnapshotErrorReport(
     SnapshotReport snapshotReport = SnapshotReportImpl.forError(snapshotContext, e);
     engine.getMetricsReporters().forEach(reporter -> reporter.report(snapshotReport));
   }
+
+  public TransactionBuilder createReplaceTableTransactionBuilder(Engine engine, String engineInfo) {
```

Comment:
Can you group this with the other public APIs in this class? i.e. not underneath the private helper methods?

Reply:
yeah good call thanks